src/issue/screens/issue.screen.js

```diff
@@ -77,7 +77,7 @@ const compareCreatedAt = (a, b) => {
 };
 
 class Issue extends Component {
-  static navigationOptions = ({ navigation }) => {
+  static navigationOptions = ({ navigation, i18n }) => {
```

Comment:
We can fix the probelm of that single `translate` in the routes file by doing it here. For example:

```
  static navigationOptions = ({ navigation, i18n }) => {
    const { state, navigate } = navigation;
    const { issue, issueURL, isPR } = state.params;

    const issueNumberRegex = /issues\/([0-9]+)(#|$)/;
    const number = issue ? issue.number : issueURL.match(issueNumberRegex)[1];
    const issueType = isPR ? i18n.t`Pull Request` : i18n.t`Issue`;

    const getHeaderIcon = () => {
      // ...
    };

    return { title: `${issueType} #${number}`, headerRight: getHeaderIcon() };
  };
```

With that being said, for some interesting reason nothing defined in the navigationOptions _here_ are rendering in this PR. For example, I don't see the cog on the right hand side where I should 🤔 

![simulator screen shot - iphone 6 - 2018-03-30 at 18 09 35](https://user-images.githubusercontent.com/12476932/38155405-a1d51b14-3445-11e8-8fff-9837cedec977.png)

Another thing I wanted to mention (and completely outside this PR and my fault 😅) is I think we need to move all screen specific route configurations to `static navigationOptions` in their component files and not in `routes.js` Things are a bit unwieldy there so I'll create a refactoring ticket where I can get to that sometime soon.

Reply:
Just noticed the navigationOptions issue. Definitely with you on use static navigationOptions()!